iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -11,6 +11,10 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
```

Comment:
Sorry, but we now need a separate line for each copyright owner. So for this file it has to be like that:
``// Copyright (c) 2020, 2021 by Robert Bosch GmbH. All rights reserved.``
``// Copyright (c) 2020, 2021 by Apex.AI Inc. All rights reserved.``

Reply:
All right good to know. Everytime we modify a filey, should we add Robert Bosch Gmbh even if we didn't create the file ?

Reply:
@JimmyBch please take a look at the Contributing.md in section `header`. Maybe @marthtz could support here. 
From my point of view the contributed tests are also fine, when this is clarified.

Reply:
I looked at it. But, it is not so clear regarding two organisations. I will talk with @marthtz.